dockerfile

```
@@ -3,7 +3,7 @@ FROM python:3-slim
 WORKDIR /usr/src/app
 
 # Install dependencies for ssdeep
-RUN apt-get update && apt-get install --no-install-recommends -y build-essential libfuzzy-dev
+RUN apt-get update && apt-get install --no-install-recommends -y build-essential libfuzzy-dev libwebp-dev
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)